selfdrive/car/volkswagen/carstate.py

```diff
@@ -49,6 +49,7 @@ def update(self, pt_cp, cam_cp, ext_cp, trans_type):
     ret.gasPressed = ret.gas > 0
     ret.brake = pt_cp.vl["ESP_05"]["ESP_Bremsdruck"] / 250.0  # FIXME: this is pressure in Bar, not sure what OP expects
     ret.brakePressed = bool(pt_cp.vl["ESP_05"]["ESP_Fahrer_bremst"])
+    ret.brakeHoldActive = bool(pt_cp.vl["ESP_05"]["ESP_Autohold_aktiv"])
```

Comment:
you need to add the signal to the can parser below. see where the rest of the ESP_05 signals are added